### ISSUE TITLE
Don't scale up image on resize if image is smaller then new width or height

### ISSUE THREAD

**Issue by jornhenkes** [toughdeveloper/oc-imageresizer-plugin#41]
As far as I can see, the resized image is always generated with the width and height set in the resize function, no matter what size the original image is. It would be great if an image isn't scaled up, when the original size of an image is smaller then the resized size. This way the quality of the image isn't compromised.

**Comment by Wachem** [comment]
+1

**Comment by Unresponsive** [comment]
Yes, this would be very useful, came here to post the same suggestion!

**Comment by maartenmachiels** [comment]
Just came across the same thing! +1

**Comment by matteotrubini** [comment]
This plugin is being migrated to core (see #55), so we are trying to figure out what issues are still relevant to invite to reopen them in core.

Issues that will remain inactive in next 30 days will be closed.

**Comment by Trysystems** [comment]
Still needed, will wait it in the core.
